penguin/system/conversation.py

```
@@ -342,6 +342,58 @@ def add_action_result(
+                        "status": status,
+                    },
+                    call_id=resolved_tool_call_id,
+                    structured_output={"tool_arguments": tool_arguments or "{}"},
+                )
+                self.session.add_tool_result_record(
+                    tool_result_record_from_tool_result(
+                        tool_result,
+                        arguments=tool_arguments or "{}",
+                    )
+                )
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)